components/email.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * Manage BuddyPress Email Post Types.
+ *
+ * @since 1.6.0
+ */
+class BPCLI_Email extends BPCLI_Component {
+	/**
+	 * Create a new email post connected to an email type.
+	 *
+	 * ## OPTIONS
+	 *
+	 * --type=<type>
+	 * : Email type for the email (should be unique identifier, sanitized like a post slug).
+	 *
+	 * [--type-description=<type-description>]
+	 * : Email type description.
+	 *
+	 * --subject=<subject>
+	 * : Email subject line. Email tokens allowed. View https://codex.buddypress.org/emails/email-tokens/ for more info.
+	 *
+	 * --content=<content>
+	 * : Email content. Email tokens allowed. View https://codex.buddypress.org/emails/email-tokens/ for more info.
+	 *
+	 * [--plain-text-content=<plain-text-content>]
+	 * : Plain-text email content. Email tokens allowed. View https://codex.buddypress.org/emails/email-tokens/ for more info.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp bp email create --type=new-event --type-description="Send an email when a new event is created" --subject="[{{{site.name}}}] A new event was created" --content="<a href='{{{some.custom-token-url}}}'></a>A new event</a> was created" --plain-text-content="A new event was created"
+	 *     Success: Email post created for type "new-event".
```

Comment:
`type` does not seem to be the right word here. In the back-end, it is called `situation`. I feel that's more intuitive, kind of giving a hint of where or when certain email will be sent.

What do you think?

Reply:
Either seem fine to me. I like that `type` is shorter :) Since this is something that won't be used often, I think it doesn't matter too much.